src/xcode/ENA/ENA/Source/Services/__tests__/Mocks/MockTestStore.swift

```diff
@@ -172,6 +171,12 @@ final class MockTestStore: Store, PPAnalyticsData {
 	// MARK: - Protocol DSCListCaching
 
 	var dscList: DSCListMetaData?
+
+	// MARK: - Protocol AppFeaturesStoring
+	#if !RELEASE
```

Comment:
I think it's not useful to have a `#if !RELEASE` condition inside of the `#if DEBUG` scope, or is it?